environment.yml

```diff
@@ -2,14 +2,15 @@ name: toffy_env
 channels:
   - conda-forge
 dependencies:
-  - python=3.8
+  - python=3.9
   - pip
   - numpy
   - seaborn
   - scikit-learn
   - jupyterlab
   - jupyter_contrib_nbextensions
   - pip:
-    - toffy
     - natsort
     - watchdog
+    - mibi-bin-tools
+    - toffy==0.1.3
```

Comment:
We need to pin mibi-bin-tools==0.2.9 and alpineer==0.1.7 right?

Reply:
@camisowers The minimum versions are set in the `pyproject.toml` file. It isn't necessary to pin them in the environment.yml. Would it be better to pin them in the `environment.yml`

Reply:
Isn't it going to keep doing the thing where it pip installs the latest version for the conda environment? 

I think for people using toffy_env it needs to be explicitly pinned. Really I'm inclined to say they should be explicitly pinned in `pyproject.toml` as well, since the multiple tiled stitching was able to pass testing when it would actually error out for toffy_env users.

Reply:
Sure, sounds good. I'll pin it in both.